views/index.ejs

```diff
@@ -67,79 +68,128 @@
   </nav>
 
   <div class="row">
-    <div class="col-md-12 m-body">
-
+    </div>
+</div>
       <div>
         <!-- Insert html view templates here -->
           <ui-view></ui-view>
 
+     
+    </div>
+  
+
+<!--_________________________________________________________________________-->
+
+  <script type="text/ng-template" id="/home.html">
+  <link rel="stylesheet"  type="text/css" href="style2.css">
+  <link rel="stylesheet"  type="text/css" href="vertical.css">
+
+<div class="container-fluid">
```

Comment:
This is to be put into the userHome.html template. 
There is no change in this template for now.